File: turtlesim_controller_py/turtlesim_controller_py/turtlesim_controller_node.py
#!/usr/bin/python3

import rclpy

from turtlesim_controller_py.turtlesim_controller import TurtlesimController

def main(args=None) -> None:
    rclpy.init(args=args)

    turtlesim_controller = TurtlesimController()

    rclpy.spin(turtlesim_controller)

    turtlesim_controller.destroy_node()
    rclpy.shutdown()

if __name__ == '__main__':
    main()

File: turtlesim_controller/src/turtlesim_controller_node.cpp
#include <rclcpp/rclcpp.hpp>

#include "turtlesim_controller/turtlesim_controller.hpp"

int main(int argc, char *argv[])
{
  rclcpp::init(argc, argv);
  rclcpp::spin(std::make_shared<TurtlesimController>());
  rclcpp::shutdown();

  return 0;
}